datahub-web-react/src/conf/Browse.tsx

```diff
@@ -1,4 +1,4 @@
 /*
     Number of results shown per browse results page
 */
-export const RESULTS_PER_PAGE = 20;
+export const RESULTS_PER_PAGE = 2;
```

Comment:
why 2?

Reply:
This is incorrect. Should match search at 10.